### PR TITLE
Added contributing guideline file

### PR DESCRIPTION
**Pull request by dlsniper**
In the spirit of Github, I think there should be a contributing file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- [IntelliJ Coding Guidelines](http://www.jetbrains.org/display/IJOS/IntelliJ+Coding+Guidelines)
+- [Contributor License Agreement](http://www.jetbrains.org/display/IJOS/Contributor+Agreement)
+- [Contribute Code](http://www.jetbrains.org/display/IJOS/Contribute#Contribute-ContributeCode)
+
+Additionally, you **NEED** to have a signed CLA sent to JetBrains before your
+code can be merged, You can download one from [Contributor License Agreement](http://www.jetbrains.org/display/IJOS/Contributor+Agreement)
+
+Building the IDE
+==
+
```
Comment (thiagokronig) [inline CONTRIBUTING.md]: ,

Comment (dlsniper) [inline CONTRIBUTING.md]: I honestly have no clue what I should do with this feedback. Please be more detailed.

**Comment by klikh** [comment]
And have you submitted a contributor license agreement as well? If not, please download the PDF from http://www.jetbrains.org/display/IJOS/Contributor+Agreement and then print it out, sign (mention you github account name there), scan and email to contribute@jetbrains.com

**Comment by dlsniper** [comment]
Hi @klikh as far as I know I've did that as I already have a Confluence account and rights to edit the IntelliJ space on it.

**Comment by klikh** [comment]
Well, if you're not a JetBrains employee and if you didn't send a scanned contribution agreement yet, you should do it. That's actually what is mentioned in the contributing.md :)

**Comment by dlsniper** [comment]
@klikh I've sent it by fax on 21.11.2014. Should I send it by mail as well?

**Comment by klikh** [comment]
It seems that we didn't receive it by fax: I've just asked our colleague to double-check, and they confirmed the fact. Could you please resend it by e-mail? Thanks.

**Comment by klikh** [comment]
The CA was received. Thank you!

**Comment by klikh** [comment]
The pull request has been merged. Thanks for your contribution.

**Comment by dlsniper** [comment]
Glad to hear, thank you! And thanks for sorting out the CLA stuff.

**Pull request closed without merging**